packages/sdk-ts/src/client/indexer/types/derivatives.ts

```diff
@@ -191,6 +192,11 @@ export interface FundingRate {
   timestamp: number
 }
 
+export interface FundingRateWithPagination {
```

Comment:
Not the pattern we do, you don't need to specify the exact interface it returns, or if you want you can just do it as 
```ts
export const transformer = (response: Response): {
  fundingRates: FundingRate[]
  pagination: ExchangePagination
} => { /* */ }
```

Reply:
@bangjelkoski got it, I destroyed this interface